src/utils/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './ui';
 export * from './validation';
 export * from './languages';
 export * from './getQuery';
+export * from './stringToDateFormat';
```

Comment:
let's rename this file to `date.ts`, we may have another date utils as well

Reply:
done